metadata.json

```diff
@@ -19,7 +19,7 @@
   "dependencies": [
     {
       "name": "puppetlabs-apt",
-      "version_requirement": ">= 2.0.0 < 3.0.0"
+      "version_requirement": ">= 2.0.0 < 5.0.0"
```

Comment:
please bump this to 6.0.0

Reply:
Done!  I hadn't spotted that 5.0.0 (and now 5.0.1) had come out :)